index.js

```diff
@@ -1,23 +1,21 @@
 'use strict';
+
+const copyProp = (to, from, prop) =>
+  Object.defineProperty(to, prop, Object.getOwnPropertyDescriptor(from, prop));
+
 module.exports = (to, from) => {
+	copyProp(to, from, 'length');
+
 	if (from.name) {
```

Comment:
`Object.getOwnPropertyNames(from)` includes `name` too, so we need to move this to after the loop.